### ISSUE TITLE
Add an export button to assignment deliveries

### ISSUE THREAD

**Issue by llaske** [llaske/sugarizer-server#366]
The idea is to add an export button in the Assignment deliveries screen to allow exporting assignments deliveries as a CSV file.

![image](https://user-images.githubusercontent.com/981841/219852957-6579d380-386a-417f-a5b4-dc3e1cf23922.png)

Only deliveries matching the filter should be exported.
You can take inspiration from the Export Users button in the Users view.

**Comment by parteekcoder** [comment]
hey @llaske @NikhilM98 I have a question that do we want to export only the assignments that are delivered right?

**Comment by llaske** [comment]
@parteekcoder no, the idea is to use filter condition so the teacher could choose.

**Comment by parteekcoder** [comment]
hey @llaske @NikhilM98 When a teacher has created an assignment how it will reach to students , @llaske on your screen above students are shown but on my screen assignments are shown
![image](https://user-images.githubusercontent.com/100110395/222973513-921db08c-c546-4bde-8ca3-d60ab40a7d74.png)
I have also added students in the class

**Comment by llaske** [comment]
The assignment should be launched (play button) to be visible by students and to have deliveries visible. See [here](https://twitter.com/sugarizerapp/status/1611754057722470401?s=46&t=bElbaEXeXEcP1TmthZV_1A) for a demo.

**Comment by parteekcoder** [comment]
Thanks @llaske for the help

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer-server/pull/396

**Comment by parteekcoder** [comment]
Hi @llaske is this fix released ?

**Comment by llaske** [comment]
@parteekcoder yes but I'm closing the issue only when it's released. it's why it's tagged "to be release".
